source/_components/netgear_lte.markdown

```diff
@@ -52,63 +60,44 @@ password:
   description: The password used for the modem web interface.
   required: true
   type: string
-{% endconfiguration %}
-
-## {% linkable_title Notifications %}
-
-The `netgear_lte` platform allows you to use a Netgear LTE modem for notifications from Home Assistant. The message will be sent as an SMS text message.
-
-```yaml
-# Example configuration.yaml entry
 notify:
-  - platform: netgear_lte
-    name: sms
-    target: "+15105550123"
-```
-
-{% configuration %}
-target:
-  description: The phone number of a default recipient or a list with multiple recipients.
-  required: true
-  type: string, list
-name:
-  description: Setting the optional parameter `name` allows multiple notifiers to be created.
+  description: A list of notification services connected to this specific host.
   required: false
-  default: notify
-  type: string
-host:
-  description: The modem to use. Not needed if you only have one.
+  type: list
+  keys:
+    recipient:
+      description: The phone number of a default recipient or a list with multiple recipients.
+      required: false
+      type: string, list
+    name:
+      description: The name of the notification service.
+      required: false
+      default: notify
+      type: string
+sensor:
+  description: Configuration options for sensors.
   required: false
-  type: string
+  type: map
+  keys:
+    monitored_conditions:
+      description: Sensor types to create.
+      required: true
```

Comment:
false

Reply:
Add `default: usage`.

Reply:
Hmm ... I would like the default to be underspecified but maybe that is not something we do?

Reply:
Why? Nah, we should be accurate in the docs.

Reply:
To discourage users from depending on it. I wanted the default to just hint that sensors are available and then users could specify the ones they actually want.

I am fine with listing the default though.